extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -90,19 +91,25 @@ describe('integration test: a4a', () => {
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     xhrMock = sandbox.stub(Xhr.prototype, 'fetch');
+    // Expect key set fetches for signing services.
+    const fetchJsonMock = sandbox.stub(Xhr.prototype, 'fetchJson');
+    for (const serviceName in signingServerURLs) {
```

Comment:
[Prefer `Object.keys` over `for`-`in` loops.](https://google.github.io/styleguide/jsguide.html#features-for-loops)